app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,6 @@
+class PostsController < ApplicationController
+  def index
+    @user = current_user
+    @posts = Post.all
```

Comment:
should we sort them at all? by created_at or something?

Reply:
👍